### PR TITLE
Fix bugs in sidebar, thresholdgraph

### LINKED ISSUE #69: Sidebar overlay not covering location input searchbar
**Describe the bug**

Screen Shot 2023-03-05 at 12 43 30 PM

**Expected behavior**
See designs

### PR DESCRIPTION
**Pull request by ericcccsliu**
## Status:
:rocket: Ready

## Description

- fix sidebar overlay bug
- fix threshold graph bug where the current time dot wasn't rendering
- fix appearance of Graph by deactivating dots 
- automatically redirect to /location from /
- hook up dashboard modules to forecast data rather than actual data 
- reorganizes modules and graphs into respective directories 

Fixes #69,

### CODE REVIEW

File: components/dashboard_modules/CurrentCapacity.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const CurrentCapacity = () => {
           : cur_output != null &&
             installed_capacity_kw != null &&
             installed_capacity_kw != 0
-          ? (cur_output / installed_capacity_kw).toFixed(0) + '%'
+          ? ((100 * cur_output) / installed_capacity_kw).toFixed(2) + '%'
```
Comment (AndrewLester) [inline components/dashboard_modules/CurrentCapacity.tsx:52]: Is cur output not in kW?

File: components/graphs/Graph.tsx [hunk 2]
```diff
@@ -72,6 +72,7 @@ const Graph = () => {
             type="monotone"
             dataKey="expected_generation_kw"
             stroke="#FFD053"
+            dot={false}
```
Comment (AndrewLester) [inline components/graphs/Graph.tsx:75]: Just like that, they're gone

**Review by AndrewLester — APPROVED** [review]

File: components/dashboard_modules/CurrentCapacity.tsx (hunk 1 above)
Comment (ericcccsliu) [reply components/dashboard_modules/CurrentCapacity.tsx:52]: it is but to get percentage we need to multiply by 100

Comment (ericcccsliu) [reply components/dashboard_modules/CurrentCapacity.tsx:52]: or we just get a cute lil decimal

**Pull request merged**